src/module/iotjs_module_i2c.cpp

```diff
@@ -45,69 +48,207 @@ I2c* I2c::GetInstance() {
   return i2c;
 }
 
+
 JHANDLER_FUNCTION(SetAddress) {
-  IOTJS_ASSERT(!"Not implemented");
+  JHANDLER_CHECK(handler.GetArgLength() == 1);
+  JHANDLER_CHECK(handler.GetArg(0)->IsNumber());
+
+  I2c* i2c = I2c::GetInstance();
+  i2c->SetAddress(handler.GetArg(0)->GetNumber());
 
   handler.Return(JVal::Null());
```

Comment:
@LaszloLango  'SetAddress' function is not for user. It is called in another functions in i2c.js file.
Also, Node.js doesn't have i2c module. I refer to 'node-i2c' module in npm.